frontend/apps/reader/modules/readerstatus.lua

```diff
@@ -113,6 +113,13 @@ function ReaderStatus:onEndOfBook()
                 },
             },
             {
+                {
+                    text = _("Move to archive"),
+                    callback = function()
+                        local MoveToArchive = require("plugins/movetoarchive.koplugin/main")
```

Comment:
I would probably prefer to do that more like:

https://github.com/koreader/koreader/blob/1e244d035832d3abd3cf27625d94aeae08c5e8e0/frontend/apps/reader/modules/readerlink.lua#L594-L602